packages/experiments-realm/product-requirement-document.gts

```diff
@@ -7,10 +7,351 @@ import {
+          </details>
+        {{/if}}
+        {{#if @model.moduleURL}}
+          <details open={{bool @model.appInstances}}>
+            <summary><span>App</span></summary>
+            <div class='details-content'>
+              <@fields.appInstances />
+              {{#if @context.actions.createCard}}
+                <Button
+                  {{on 'click' this.createInstance}}
+                  class='generate-button new-instance-button'
+                  @kind='primary-dark'
+                  @disabled={{this._createInstance.isRunning}}
+                  @loading={{this._createInstance.isRunning}}
+                >
+                  {{#unless this._createInstance.isRunning}}
+                    <span class='generate-button-logo' />
+                  {{/unless}}
+                  Create New Instance
+                </Button>
+              {{/if}}
+            </div>
+          </details>
+        {{/if}}
+
+        <details open={{bool @model.prompt}}>
+          <summary><span>Prompt</span></summary>
+          <div class='details-content'>
+            <@fields.prompt />
+          </div>
+        </details>
+        <details open={{bool @model.overview}}>
+          <summary><span>Overview</span></summary>
+          <div class='details-content'>
+            <@fields.overview />
+          </div>
+        </details>
+        <details open={{bool @model.schema}}>
+          <summary><span>Schema</span></summary>
+          <div class='details-content'>
+            <@fields.schema />
+          </div>
+        </details>
+        <details open={{bool @model.layoutAndNavigation}}>
+          <summary><span>Layout & Navigation</span></summary>
+          <div class='details-content'>
+            <@fields.layoutAndNavigation />
+          </div>
+        </details>
+      </div>
+    </section>
+    <style>
+      .prd {
+        padding: var(--boxel-sp) var(--boxel-sp-xxl);
+      }
+      .title-group {
+        display: flex;
+        align-items: center;
+        gap: var(--boxel-sp);
+      }
+      .header-button-group {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        gap: var(--boxel-sp);
+      }
+      .generate-button {
+        --icon-size: 20px;
+        --boxel-button-loading-icon-size: var(--icon-size);
+        padding: var(--boxel-sp-xxs) var(--boxel-sp);
+        justify-self: end;
+        gap: var(--boxel-sp-sm);
+        white-space: normal;
+      }
+      .generate-button :deep(svg) {
+        width: var(--icon-size);
+        height: var(--icon-size);
+      }
+      .generate-button :deep(.loading-indicator) {
+        width: var(--icon-size);
+        height: var(--icon-size);
+        margin-right: 0;
+      }
+      .generate-button-logo {
+        flex-shrink: 0;
+        display: inline-block;
+        width: var(--icon-size);
+        height: var(--icon-size);
+        background: url('./ai-assist-icon@2x.webp') no-repeat center;
```

Comment:
I see this was already in the styles but when I ran this locally the image was missing 😯

Reply:
<img width="834" alt="Boxel 2024-08-27 17-30-00" src="https://github.com/user-attachments/assets/dd860431-197c-4002-b216-5946d5551051">

Reply:
@backspace currently it only appears when using localhost 4201. Probably a good idea to move this icon to boxel-ui and use it from there. Do you know a way to import/reference icon urls from boxel-ui for use in css `background-image` property? I generally end up changing it to not be a css property and use the icon directly in the template instead.

Reply:
hmm yes this is confusing, since [this PR](https://github.com/cardstack/glimmer-scoped-css/pull/24) I would expect `glimmer-scoped-css` to be able to handle a reference in the CSS to an asset but when I tried that just now it broke the build 🧐

<img width="513" alt="bm2 (Git) 2024-08-28 15-34-49" src="https://github.com/user-attachments/assets/f0369b51-e93a-40c5-ba95-25b98a5417eb">

<img width="847" alt="pnpm start 2024-08-28 15-35-17" src="https://github.com/user-attachments/assets/eb3e1e99-cfc6-48ac-b86f-789026a5218f">

will examine this further